pkg/csi/service/wcpguest/controller.go

```diff
@@ -126,6 +129,13 @@ func (c *controller) Init(config *commonconfig.Config, version string) error {
 		return err
 	}
 
+	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.WorkloadDomainIsolation) {
```

Comment:
We can not check WCP capability in PVCSI.
We need to introduce new FSS for PVCSI changes.

Reply:
replacing this with PVCSI FSS
`if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.WorkloadDomainIsolationFSS) {`